setup.cfg

```diff
@@ -3,4 +3,56 @@ version_variable=teslajsonpy/__version__.py:__version__
 upload_to_pypi=false
 check_build_status=false
 remove_dist=false
-hvcs=github
\ No newline at end of file
+hvcs=github
+
+[pydocstyle]
+ignore = D202, D212, D416, D213, D203, D407
+
+[flake8]
+exclude = .venv,.git,.tox,docs,venv,bin,lib,deps,build
+# To work with Black
+max-line-length = 88
+# E501: line too long
```

Comment:
Let's increase max-line-length to 120 instead of ignoring E501

Reply:
I'm just taking the HA ignores for now. They default to 88 since they're using black.

Reply:
We don't need to keep HA code-style in our repo as long as it separated library.

Reply:
True, but remember one of the goals #24 was to potentially move some of the HA-specific code over to HA? It'd make it easier if we enforced similar code-styles.

I'm pretty sure black would ignore the 120 length anyway and bring it back down to 88.

Reply:
@alandtse ok.